src/sdc11073/consumer/consumerimpl.py

```diff
@@ -424,6 +426,7 @@ def subscription_mgr(self) -> ConsumerSubscriptionManagerProtocol:
     def start_all(self, not_subscribed_actions: list[str] | None = None,  # noqa: PLR0913
                   fixed_renew_interval: float | None = None,
                   subscribe_periodic_reports: bool = False,
+                  subscribe_system_error_report: bool = True,
```

Comment:
list instead of each single report?

Reply:
updated start_all Method, now only the list not_subscribed_actions is relevant. Both bool parameters removed.